src/components/fields/select-field/select-field.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import requiredIf from 'react-required-if';
+import Constraints from '../../constraints';
+import Spacings from '../../spacings';
+import FieldLabel from '../../field-label';
+import SelectInput from '../../inputs/select-input';
+import createSequentialId from '../../../utils/create-sequential-id';
+import filterDataAttributes from '../../../utils/filter-data-attributes';
+import FieldErrors from '../../field-errors';
+
+const sequentialId = createSequentialId('text-field-');
+
+const hasErrors = errors => errors && Object.values(errors).some(Boolean);
+
+export default class SelectField extends React.Component {
+  static displayName = 'SelectField';
+
+  static isTouched = SelectInput.isTouched;
+
+  // customizable components
+  static ClearIndicator = SelectInput.ClearIndicator;
+  static Control = SelectInput.Control;
+  static DropdownIndicator = SelectInput.DropdownIndicator;
+  static Group = SelectInput.Group;
+  static GroupHeading = SelectInput.GroupHeading;
+  static IndicatorsContainer = SelectInput.IndicatorsContainer;
+  static IndicatorSeparator = SelectInput.IndicatorSeparator;
+  static Input = SelectInput.Input;
+  static LoadingIndicator = SelectInput.LoadingIndicator;
+  static Menu = SelectInput.Menu;
+  static MenuList = SelectInput.MenuList;
+  static LoadingMessage = SelectInput.LoadingMessage;
+  static NoOptionsMessage = SelectInput.NoOptionsMessage;
+  static MultiValue = SelectInput.MultiValue;
+  static MultiValueContainer = SelectInput.MultiValueContainer;
+  static MultiValueLabel = SelectInput.MultiValueLabel;
+  static MultiValueRemove = SelectInput.MultiValueRemove;
+  static Option = SelectInput.Option;
+  static Placeholder = SelectInput.Placeholder;
+  static SelectContainer = SelectInput.SelectContainer;
+  static SingleValue = SelectInput.SingleValue;
+  static ValueContainer = SelectInput.ValueContainer;
+
+  static propTypes = {
+    // SelectField
+    id: PropTypes.string,
+    horizontalConstraint: PropTypes.oneOf(['xs', 's', 'm', 'l', 'xl', 'scale']),
+    errors: PropTypes.shape({
+      missing: PropTypes.bool,
+    }),
+    renderError: PropTypes.func,
+    isRequired: PropTypes.bool,
+    touched: (props, ...rest) =>
+      props.isMulti
+        ? PropTypes.arrayOf(PropTypes.bool, ...rest)(props, ...rest)
+        : PropTypes.bool(props, ...rest),
+
+    // SelectInput
+    'aria-label': PropTypes.string,
+    'aria-labelledby': PropTypes.string,
+    isAutofocussed: PropTypes.bool,
```

Comment:
huh... [guess I was the confused one](http://www.future-perfect.co.uk/grammar-tip/is-it-focussed-or-focused/)